src/main/java/gift/ProductAdminController.java

```diff
@@ -0,0 +1,61 @@
+package gift;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/products")
```

Comment:
Admin이라는걸 나타내기 위해
```suggestion
@RequestMapping("admin/products")
```
이렇게 한다면 어드민과 프로덕션과 주소를 보면 분리해볼수 있지 않을까요?